download_convert_train.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+### Configs
+
+#DATA_DIR=./coins-5-classes
+#DATASET_URL=https://s3.amazonaws.com/coin-vision/coins-5-classes.csv
+
+DATASET_DIR=./usa-coins-398
+DATASET_URL=https://s3.amazonaws.com/coin-vision/usa-coins-398-classes.csv
+
```

Comment:
we should use something like version of dataset - `/v2/` or `2018-12-3` - in case it will be updated